app/index.rounded.html

```diff
@@ -23,6 +23,7 @@
   </style> 
   </head>
   <body>
+    <z-favicon></z-favicon>

```

Comment:
With the requested changes to the component, I think the z-favicon element should be added to the head where a favicon would normally be.

Reply:
Removed as we decided to remove `<z-favicon>` element.